server/sockets/list.js

```diff
@@ -143,63 +146,48 @@ const updateItemState = (socket, itemClientLocks) => {
 
 const updateItem = (
   io,
-  dashboardViewClients,
-  cohortViewClients,
-  listClients,
-  viewersIds
+  cohortClients,
+  dashboardClients,
+  listClients
 ) => data => {
-  const { listId, userId, itemId } = data;
-  const sanitizedUserId = sanitize(userId);
-
-  return User.findById(sanitizedUserId).then(user => {
```

Comment:
does it update edited by property on front end now??

Reply:
Yes it updates this prop